src/sai/Assignment15/Accounts.java

```diff
@@ -0,0 +1,149 @@
+package sai.Assignment15;
+
+/*
+ *Assignment - 15 : 9th April'2021
+ *
+ * A person has total 4 accounts in ICICI, HDFC, BOB, SBI one in each.
+a) Person can debit or credit amount from respective bank.
+Hint : Switch case is expected here. [Please watch today's recording again, in case of confusion.]
+
+b) A Method to show total balance [ICICI balance + HDFC balance + BOB balance + SBI Balance]
+
+c) A Method to show total credit, debit operations
+output : Total credit operation - 7 times
+Total debit operation - 5 times
+
+d) A Method to show individual bank credit operations.
+output : ICICI bank Credit operation - 2
+HDFC bank Credit operation - 2
+SBI bank Credit operation - 1
+BOB bank Credit operation - 4
+
+e) A Method to show individual bank debit operations.
+output : ICICI bank Debit operation - 2
+HDFC bank Debit operation - 2
+SBI
+
+ */
+public class Accounts {
+
+	int iciciBalance = 125000, hdfcBalance = 150000, bobBalance = 175000, sbiBalance = 185000;
+	int iciciCreditCount = 0, hdfcCreditCount = 0, bobCreditCount = 0, sbiCreditCount = 0, totalCreditCount = 0;
+	int iciciDebitCount = 0, hdfcreditCount = 0, bobDebitCount = 0, sbiDebitCount = 0, totalDebitCount = 0;
+
+	void debitCredit(String bank, String operation, int amt) {
+		switch (operation) {
+		case "debit":
+			switch (bank) {
+			case "icici":
+				if (iciciBalance >= amt) {
```

Comment:
instead of writing same code block for multiple times, you can write debit() method which accept the two arguments and you can simply call that method.

Code snippet

int debit(int totalAmt, int debitAmt) {
		if (totalAmt < debitAmt) {
			return 0;
		} else
			return totalAmt - debitAmt;
}

totalAmt will your iciciAmt (instance variable)